test/inductor/test_cudagraph_trees.py

```diff
@@ -2445,6 +2445,15 @@ def iter(batch_size: int, mod: torch.nn.Module):
                 exactly=True,
             ).run("\n".join(captured_output))
 
+        @torch._inductor.config.patch("cpp_wrapper", 1)
+        def test_cpp_wrapper(self):
+            def f(x):
+                return torch.sin(x)
+
+            compiled = torch.compile(f, mode="reduce-overhead")
+            print(compiled(torch.randn(10, device="cuda")))
```

Comment:
Use `run_twc`? and check `assertEqual` to eager results ? also, should not print in test.